packages/components/src/DateTimePickers/pickers/TimePicker/TimePicker.component.test.js

```diff
@@ -41,25 +41,5 @@ describe('TimePicker component', () => {
 					.hasClass('highlight'),
 			).toBe(true);
 		});
-		it('should scroll the first match into view when user inputs', () => {
```

Comment:
Remove this test case because the `componentWillUpdate` has been removed here:
https://github.com/Talend/ui/pull/2860/files#diff-45f8e95db6daaa9d3e4125618e1d07bfL79